Description: add printing support
 .
 handylinux-thunar-actions (0.3-3) unstable; urgency=medium
 .
   * modification du script pixup par thuban
Author: arnault perret <arpinux@member.fsf.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- handylinux-thunar-actions-0.3.orig/scripts/imagemod-hl
+++ handylinux-thunar-actions-0.3/scripts/imagemod-hl
@@ -1,6 +1,7 @@
 #!/bin/sh
 # choose how to modify an image
-# requires zenity and imagemagick
+# requires zenity and imagemagick 
+# and photoprint 
 
 # action personnalisée dans thunar : 
 # imagemod-hl.sh %F
@@ -19,6 +20,7 @@ TOGIF="Convert to GIF"
 RESIZE="Resize"
 TURNR="Turn right"
 TURNL="Turn left"
+PRINTIMG="Print images"
 
 case $LANG in
 	fr*)
@@ -31,6 +33,7 @@ case $LANG in
         RESIZE="Redimensionner l'image"
         TURNR="Tourner à droite"
         TURNL="Tourner à gauche"
+        PRINTIMG="Imprimer les images"
         ;;
     de*)
 		scaletitle="Stufenbild"
@@ -65,6 +68,7 @@ R=$(zenity --list --text "$scaletext"\
     FALSE "4: $RESIZE" \
     FALSE "5: $TURNR" \
     FALSE "6: $TURNL"\
+    FALSE "7: $PRINTIMG"\
     | cut -d':' -f1)
 
 case $R in 
@@ -101,6 +105,8 @@ case $R in
             convert -rotate -90 $i $i
         done
         ;;
+    7 )
+        photoprint $@ ;;
 esac
     
 IFS=$OLDIFS
--- handylinux-thunar-actions-0.3.orig/uca/uca.xml
+++ handylinux-thunar-actions-0.3/uca/uca.xml
@@ -78,7 +78,7 @@
 	<name>Modifier l&apos;image</name>
 	<unique-id>1440701060510825-4</unique-id>
 	<command>/usr/bin/imagemod-hl %F</command>
-	<description>pivoter, redimensionner, convertir une image</description>
+	<description>pivoter, redimensionner, convertir ou imprimer une image</description>
 	<patterns>*</patterns>
 	<startup-notify/>
 	<image-files/>
--- handylinux-thunar-actions-0.3.orig/uca/uca.xml.en
+++ handylinux-thunar-actions-0.3/uca/uca.xml.en
@@ -78,7 +78,7 @@
 	<name>Modify image</name>
 	<unique-id>1440701060510825-4</unique-id>
 	<command>/usr/bin/imagemod-hl %F</command>
-	<description>rotate, scale, convert an image</description>
+	<description>rotate, scale, convert or print an image</description>
 	<patterns>*</patterns>
 	<startup-notify/>
 	<image-files/>
